packages/fields/src/fields/title/view.tsx

```diff
@@ -43,19 +44,29 @@ const TitleView = ( { item }: { item: BasePost } ) => {
 			</span>
 		);
 	}
+	return <BaseTitleView item={ item }>{ suffix }</BaseTitleView>;
+}
 
+export function BaseTitleView( {
+	item,
+	children,
+}: {
+	item: CommonPost;
+	children?: ReactNode;
+} ) {
+	const renderedTitle = getItemTitle( item );
 	return (
 		<HStack
 			className="edit-site-post-list__title"
 			alignment="center"
 			justify="flex-start"
 		>
-			<span>
-				{ decodeEntities( renderedTitle ) || __( '(no title)' ) }
-			</span>
-			{ suffix }
+			<span>{ renderedTitle || __( '(no title)' ) }</span>
+			{ children }
 		</HStack>
 	);
-};
+}
 
-export default TitleView;
+export default function TitleView( { item }: { item: CommonPost } ) {
```

Comment:
Can't we use `BaseTitleView` directly in the fields? Why to create a wrapper component?

Reply:
Because these functions are passed to the field's `render` function with a different signature. It doesn't make sense to me to change the `render` signature to have `children` too.